model-optimizer/extensions/ops/priorbox.py

```diff
@@ -31,6 +32,7 @@ def __init__(self, graph: Graph, attrs: dict):
             'op': self.op,
             'version': 'opset1',
             'flip': 1,
+            'scale_all_sizes': 1,
```

Comment:
`True`? Update `flip` and `clip` as well

Reply:
Done.